README.md

```diff
@@ -1,2 +1,24 @@
-# palindrome-checker
-first project of Java-script.
+# Palindrome-checker 🚀
+
+## Project Description 📝
+
+
+- 1sr project of javascript.
+- Its a palidrome cheker, it shows if a word is palidrome or not.
+- learnt new js,html and css codes and features.
+
+## Technologies Used 🛠️
+
+
+- HTML
+- CSS
+- JavaScript
+
+
+## Author 👩‍💻
+
+- LinkedIn: https://www.linkedin.com/in/sumita-balouch-959197309?utm_source=share&utm_campaign=share_via&utm_content=profile&utm_medium=ios_app
+- Email: sumitabalouch@gmail.com
+
+## License 📜
+MIT license.
```

Comment:
- Add the Demo section with a screenshot and also add the Usage or How to use section and give users an instruction on how they can use your project.